language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/components/BotTest.scala

```diff
@@ -229,24 +230,24 @@ final class BotTest extends FlatSpec with Matchers with Eventually {
       }
 
     val counter = new AtomicInteger(0)
-    Bot.wire(appId, ledgerClient, transactionFilter, bot, _ => counter)
-
-    // when the bot is wired-up, no command should have been submitted to the server
-    ledgerClient.submitted should have size 0
-    counter.get shouldBe 0
+    using(Bot.wire(appId, ledgerClient, transactionFilter, bot, _ => counter)) { _ =>
```

Comment:
Not to be picky, however it seems you don't use the return of the `Bot.wire`. Is that by mistake? If not, you can simplify the `using`.

Reply:
Yup. I guess today I'm over-engineering a lot. Thank you for pointing it out!